themes/default/content/blog/announcing-crossguard-preview/index.md

```diff
@@ -82,7 +82,7 @@ const bucket = new aws.s3.Bucket("my-bucket", {
 });
 ```
 
-![Demo of Policy as Code](pac-demo.gif)
+![Demo of Policy as Code](pac-demo.webp)
```

Comment:
Looks like this file is actually much larger as `webp`
<img width="910" alt="Screen Shot 2022-11-10 at 4 40 36 PM" src="https://user-images.githubusercontent.com/24326455/201235478-7a845129-e31e-4c0d-a4ac-4d482c44f4e4.png">